api/src/chat/localization/RcdaTextEnglish.ts

```diff
@@ -16,6 +16,8 @@ export class RcdaCommonTextEnglish {
     selectDropdownPlaceholder = "Choose...";
     yes = "Yes";
     no = "No";
+    // TODO Chane the URL for the supported channel.
+    unsupportedChannel = "We currently do not support this channel. Please visit us at - https://esthumstrbotstg01.z13.web.core.windows.net/?#/chat";
```

Comment:
Let's use hyperlink text for the action to come to the website. Let's also reference the link out of an environment variable. We probably also want to keep it out of the localizer. If it needs to appear in the text message we should opt for passing it in as an argument. Instead of that, though, let's use a 'Hero Card' maybe, or something is presented like an action or a button and is widely supported on all platforms.

Reply:
Added a HeroCard with localization.